Gemfile

```diff
@@ -0,0 +1,74 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '3.1.3'
+
+# Bundle edge Rails instead: gem "rails", github: "rails/rails", branch: "main"
+gem 'rails', '~> 7.0.4', '>= 7.0.4.3'
+
+# The original asset pipeline for Rails [https://github.com/rails/sprockets-rails]
+gem 'sprockets-rails'
+
+# Use sqlite3 as the database for Active Record
+gem 'sqlite3', '~> 1.4'
+
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/235438540-df996ca9-ab28-4064-81bc-17e467ae0b8e.png)

- [ ] In this project, you are supposed to have PostgreSQL instead of sqlite3 😨 as you can see in the project requirements above ⬆️ kindly replace the sqlite3 with Postgresql in your project 👍 to follow the project requirements 😊